ClientProxy.go

```diff
@@ -38,10 +38,7 @@ func _D(fmt string, v ...interface{}) {
 // Probably the first is the easiest.
 
 func (this ClientProxy) ServeDNS(w dns.ResponseWriter, request *dns.Msg) {
-	proxy_req := *request
-	request_bytes, err := proxy_req.Pack()
-	// why not just pack from request directly? (I haven't tried this...)
-	//request_bytes, err := reques.Pack()
+	request_bytes, err := request.Pack() //I am not sure it is better to pack directly or using a pointer
```

Comment:
There is no benefit to assigning to another variable first. It will mean copying the memory from `request` to a new location, then doing the `Pack()` operation on that. Since we don't modify the data at all, this is just an unnecessary extra step.

Go ahead and remove the comment. :)